### ISSUE TITLE
Shopping list allow optional details 

### ISSUE THREAD

**Issue by strider72** [estep248/Pocket-Plan#9]
Best described with an example.  You have "Bread" in the built-in list, with the appropriate aisle/section associated with it.  But I can't specify a type of bread, or anything specific.

What you could do is allow me to put further text in parentheses.  For example:

* Bread (whole wheat)

The app will ignore the text in the parentheses and treat it as though I'd just typed "bread", and use the category associated with "bread".

Currently, if I type the above, example, it's not on the list exactly so it doesn't recognize a category.  This feature would allow you to keep the built-in list reasonably simple.

ALSO: I haven't used it enough, but does the app learn from what I enter?

**Comment by strider72** [comment]
Another good example is:

* Vitamins (D)
* Vitamins (Flintstone)
* Vitamins (men's multi)

Maybe even let the use specify the delineator character — in this case "("

**Comment by JakeGonder** [comment]
> does the app learn from what I enter?

Yes! Right now if you type in Bread (whole weat) it wont recognize the category, but you can then choose the category and this decision will be remembered, next time you enter it, it will be recommended as an option, and if you select it, it will have the correct category. 

The parantheses sounds like a good idea, an alternative that comes to my mind would be just recommending the category that contains the best match. E.g. you enter "Bread (Whole Wheat)" => Best match is the item "Bread" in category "Bread & Pastries" => "Bread (Whole Wheat)" Is assigned to this category

**Comment by strider72** [comment]
Yes, a "closest match" guess would probably work pretty well also.  The danger of saving everything I ever enter is getting the list cluttered over time (especially typo versions)

**Comment by JakeGonder** [comment]
You can remove items the list remembered (e.g. if you entered a typo) by going to settings => shopping => manage custom items

**Comment by JakeGonder** [comment]
This feature is now live in the 1.3.1 release
